src/mono/wasm/build/WasmApp.targets

```diff
@@ -154,7 +154,19 @@
     <WasmStripILAfterAOT Condition="'$(WasmStripILAfterAOT)' == ''">false</WasmStripILAfterAOT>
 
     <WasmRuntimeAssetsLocation Condition="'$(WasmRuntimeAssetsLocation)' == ''">_framework</WasmRuntimeAssetsLocation>
-    <BlazorCacheBootResources Condition="'$(BlazorCacheBootResources)' == ''">true</BlazorCacheBootResources>
+
+    <!-- Runtime feature defaults to trim unnecessary code -->
```

Comment:
We should move this to a separate file, and share that between WasmApp.targets, and *Browser.targets, so they don't go out of sync.
Later we'll figure out the two are going to come together.

Reply:
We could move them to workload manifest, right?
Or is moving them to a common file and sharing the file easy enough? I would like to unify the bunding early in 9 cycle

Reply:
I don't think the manifest would be a good place for this. We should use a common file for now, and then see how the unification shapes up, which would give clarity on this.

Reply:
Just curiosity, why isn't the workload manifest good place?
What alternative do you suggest? Should we link the file to SDK pack and workload include it in both places?

Reply:
> Just curiosity, why isn't the workload manifest good place?

The manifest is not just for wasm, and it doesn't really belong there. But have it there for now. We'll have some refactoring happening soon anyway.